presto-parser/src/main/java/com/facebook/presto/sql/SqlFormatter.java

```diff
@@ -513,8 +519,14 @@ protected Void visitExcept(Except node, Integer indent)
             processRelation(node.getLeft(), indent);
 
             builder.append("EXCEPT ");
-            if (!node.isDistinct()) {
-                builder.append("ALL ");
+            if (node.isDistinct().isPresent()) {
```

Comment:
We normally prefer functional interface when the logic is simple. So this can be rewritten to: `node.isDistinct().map(distinct -> distinct ? builder.append("DISTINCT ") : builder.append("ALL "));`

Reply:
Alternatively you can put this in a separate helper function to reduce code duplication.

Reply:
I will look into it and update the diff.